<div style="font-family:  Arial, Helvetica Neue, Verdana, Geneva, sans-serif !important;">
<p style="text-align:center">
<h2>ePlant</h2>
<h5>ePlant is a data visualization tool for exploring and integrating multiple levels of plant data.</h5></p>

<!-- plant in cup image -->
<p style="text-align:center"><img src="http://bar.utoronto.ca/~eplant/img/plantInCupSmall.png"></p>

<p>ePlant combines new and familiar biological data visualization tools into a single portal that was designed to help researchers explore genes of interest from multiple layers of analysis. ePlant uses a zoomable user interface to help visualize the natural connections between DNA sequences, protein-protein interactions, cell and tissue expression patterns, natural variation, and proteomic responses to various experimental conditions. Dynamic links to the underlying data  and citation information is available on demand.
</p>

<p>Originally developed for the <a href="http://www.arabidopsis.org/portals/masc/IAIC.jsp">Arabidopsis Information Portal</a>, ePlant was built to be species agnostic. Please contact us if there is a particular species you would like to add. </p>

<p>ePlant was developed by <a href="http://www.waese.com">Jamie Waese</a>, Hans Yu, Asher Pasha and <a href="http://csb.utoronto.ca/faculty/provart-nicholas">Nicholas Provart</a> in the Department of Cell & Systems Biology at the University of Toronto. ePlant was funded with grants from Genome Canada and NSERC. This is a continuation of the original ePlant project by <a href="http://www.plosone.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0015237&representation=PDF">Fucile et al. (2011)</a>.</p>

<p>ePlant was designed to work with Firefox, Chrome and Safari. Some features do not work with Internet Explorer at this time.</p>


<!-- logos at bottom of page -->
<p>&nbsp;</p>
<p style="text-align:center">
<a href="http://www.csb.utoronto.ca/"><img src="img/csblogo.png" width="134" height="60" alt="Cell &amp; Systems Biology" longdesc="http://www.csb.utoronto.ca/" /></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.cagef.utoronto.ca/"><img src="http://bar.utoronto.ca/Cagef_logo.gif" width="87" height="57" /></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.utoronto.ca"><img src="img/logoUofT.gif" width="180" height="65" alt="University of Toronto" longdesc="http://www.utoronto.ca/" /></a> 
</p>
<br><br>
<h2>Background</h2>
<p>ePlant is an effort to leverage current thinking about data visualization and user experience design to help educators and researchers make sense of the ever growing volumes of publicly available data describing multiple levels of biological processes. ePlant connects to several web services to download the latest genome, proteome, metabolome, biochemical pathways, and tissue expression pattern data for any given gene.</p>

<p>The visualization tools in ePlant are presented with a conceptual hierarchy that progresses from big to small. We use a zoom metaphor to represent moving closer to- or further away from each level of analysis in the hopes that the temporal and spatial connection between layers will reveal conceptual connections as well. We recognize that each level of analysis is unique and arguments can be made for organizing the hierarchy in a different order. Future versions of ePlant will enable users to reorder the icons according to their personal preferences.</p>

<p>Some of ePlant's visualization tools are implementations of other tools you may be familiar with: <a href="http://cytoscape.github.io/cytoscape.js/">Cytoscape.js</a> for the protein interactions and reactome charts, <a href="http://wiki.jmol.org/index.php/Jmol_JavaScript_Object">JSmol</a> for the 3D molecular view, and <a href="http://jbrowse.org/">JBrowse</a> for the genome browser. Other visualization tools are based on the well known viewers developed at <a href="http://bar.utoronto.ca">bar.utoronto.ca</a>, particularly the eFP browsers for <a href="http://bar.utoronto.ca/cell_efp/cgi-bin/cell_efp.cgi">cells</a>, <a href="http://bar.utoronto.ca/efp/cgi-bin/efpWeb.cgi">tissues</a>, and <a href="http://bar.utoronto.ca/efp/cgi-bin/efpWeb.cgi?dataSource=Natural_Variation">natural variations</a>.</p>

<p>ePlant is intended to be open source and open ended. ePlant is licensed under a Creative Commons Attribution-Share Alike 2.5 Canada License. The source code is posted on <a href="https://github.com/jamiewaese/ePlant">GitHub</a>.</p>   
</div>